media-libs/opencv/opencv-4.10.0.ebuild

```diff
@@ -0,0 +1,1061 @@
+
+	mkdir -p "${S}/.cache/ade" || die
+	cp \
+		"${DISTDIR}/ade-${ADE_PV}.tar.gz" \
+		"${S}/.cache/ade/$(md5sum "${DISTDIR}/ade-${ADE_PV}.tar.gz" | cut -f 1 -d " ")-v${ADE_PV}.tar.gz" || die
+
+	if use dnnsamples; then
+		mkdir -p "${S}/.cache/wechat_qrcode" || die
+		for file in "detect.caffemodel" "detect.prototxt" "sr.prototxt" "sr.caffemodel"; do
+			mv \
+				"${WORKDIR}/${PN}_3rdparty-${QRCODE_COMMIT}/${file}" \
+				"${S}/.cache/wechat_qrcode/$( \
+					md5sum "${WORKDIR}/${PN}_3rdparty-${QRCODE_COMMIT}/${file}" | cut -f 1 -d " " \
+				)-${file}" || die
+		done
+
+		mv \
+			"${WORKDIR}/${PN}_3rdparty-${DNN_SAMPLES_FACE_DETECTOR_COMMIT}/res10_300x300_ssd_iter_140000.caffemodel" \
+			"${S}/samples/dnn/" || die
+	fi
+
+	if use contribxfeatures2d; then
+		cp \
+			"${WORKDIR}/${PN}_3rdparty-${XFEATURES2D_BOOSTDESC_COMMIT}/"*.i \
+			"${WORKDIR}/${PN}_contrib-${PV}"/modules/xfeatures2d/src/ || die
+		mkdir -p "${S}/.cache/xfeatures2d/boostdesc" || die
+		for file in "${WORKDIR}/${PN}_3rdparty-${XFEATURES2D_BOOSTDESC_COMMIT}/"*.i; do
+			mv \
+				"${WORKDIR}/${PN}_3rdparty-${XFEATURES2D_BOOSTDESC_COMMIT}/$(basename "${file}")" \
+				"${S}/.cache/xfeatures2d/boostdesc/$( \
+					md5sum "${WORKDIR}/${PN}_3rdparty-${XFEATURES2D_BOOSTDESC_COMMIT}/$(basename "${file}")" | cut -f 1 -d " " \
+				)-$(basename "${file}")" || die
+		done
+
+		cp \
+			"${WORKDIR}/${PN}_3rdparty-${XFEATURES2D_VGG_COMMIT}/"*.i \
+			"${WORKDIR}/${PN}_contrib-${PV}"/modules/xfeatures2d/src/ || die
+		mkdir -p "${S}/.cache/xfeatures2d/vgg" || die
+		for file in "${WORKDIR}/${PN}_3rdparty-${XFEATURES2D_VGG_COMMIT}/"*.i; do
+			mv \
+				"${WORKDIR}/${PN}_3rdparty-${XFEATURES2D_VGG_COMMIT}/$(basename "${file}")" \
+				"${S}/.cache/xfeatures2d/vgg/$( \
+					md5sum "${WORKDIR}/${PN}_3rdparty-${XFEATURES2D_VGG_COMMIT}/$(basename "${file}")" | cut -f 1 -d " " \
+				)-$(basename "${file}")" || die
+		done
+	fi
+
+	if use contribdnn; then
+		mkdir -p "${S}/.cache/data" || die
+		mkdir -p "${WORKDIR}/${PN}_extra-${PV}/testdata/cv/face/" || die
+		file="face_landmark_model.dat"
+		cp \
+			"${WORKDIR}/${PN}_3rdparty-${FACE_ALIGNMENT_COMMIT}/${file}" \
+			"${WORKDIR}/${PN}_extra-${PV}/testdata/cv/face/" \
+			|| die
+		mv \
+			"${WORKDIR}/${PN}_3rdparty-${FACE_ALIGNMENT_COMMIT}/${file}" \
+			"${S}/.cache/data/$( \
+				md5sum "${WORKDIR}/${PN}_3rdparty-${FACE_ALIGNMENT_COMMIT}/${file}" | cut -f 1 -d " " \
+			)-${file}" || die
+	fi
+
+	if use cuda; then
+		mkdir -p "${S}/.cache/nvidia_optical_flow"
+		cp \
+			"${DISTDIR}/NVIDIAOpticalFlowSDK-${NVIDIA_OPTICAL_FLOW_COMMIT}.tar.gz" \
+			"${S}/.cache/nvidia_optical_flow/$( \
+				md5sum "${DISTDIR}/NVIDIAOpticalFlowSDK-${NVIDIA_OPTICAL_FLOW_COMMIT}.tar.gz" | cut -f 1 -d " " \
+			)-${NVIDIA_OPTICAL_FLOW_COMMIT}.tar.gz" || die
+	fi
+
+	if use java; then
+		java-pkg-opt-2_src_prepare
+
+		JAVA_ANT_ENCODING="iso-8859-1"
+		# set encoding so even this cmake build will pick it up.
+		export ANT_OPTS+=" -Dfile.encoding=iso-8859-1"
```

Comment:
`iso-8859-1`, if still relevant, could be added as options behind the `eant` call in `src_compile()`, example https://github.com/gentoo/gentoo/blob/master/dev-embedded/arduino/arduino-1.8.19-r1.ebuild#L75

Reply:
Done in c0b97f4